### DISCUSSION TITLE
Error --- npm run build --------npm-start---->webpack

### DISCUSSION THREAD

**Discussion by evertonProgramadorCriativo** [frontendbr/forum#1579] [Perguntas e Respostas]
eu  ja fiz de tudo , ja baixei  BuildTools_Full , ja renicie o pc  e essta dando esse error  quando eu dou npm run build

estou usando node v10.16.3

-----------------------------------------------/-------------------------------------------------
Projeto link ----> https://github.com/evertonProgramadorCriativo/bootstrapGaleria
--------------------------------------------------------/-----------------------------------------

![web1](https://user-images.githubusercontent.com/55323538/67578877-bdad3900-f719-11e9-8187-d085b399c64d.png)
![web2](https://user-images.githubusercontent.com/55323538/67578878-be45cf80-f719-11e9-92b1-e1cfc362c93e.png)
![web3](https://user-images.githubusercontent.com/55323538/67578879-be45cf80-f719-11e9-9214-74cd227d8c57.png)
![web4](https://user-images.githubusercontent.com/55323538/67578881-be45cf80-f719-11e9-819a-a7abfb27c902.png)
![web5](https://user-images.githubusercontent.com/55323538/67578882-bede6600-f719-11e9-9e3e-be93735892ed.png)

**Comment by simaodeveloper** [comment]
Parece que você copiou e colou o css de outra formatação e ele não esta reconhecendo o caracter inválido

**Comment by ninetails** [comment]
aparentemente falta configurar o loader para fontes

**Comment by danilovictor** [comment]
Estou com este mesmo problema. Alguem tem alguma solução ou porque ta assim?

**Reply by ninetails** [reply]
Posta teu `webpack.config.js`. Pode ser que a ordem dos loaders pro SASS esteja incorreta (pelo menos no caso da _thread_ original, pareceu ser isso ou talvez configurar o loader pra fontes)

**Comment by danilovictor** [comment]
![image](https://user-images.githubusercontent.com/64422090/106046017-7a7e0a80-60c0-11eb-9e62-f191ef09d57d.png)

**Comment by danilovictor** [comment]
![image](https://user-images.githubusercontent.com/64422090/106046300-d5affd00-60c0-11eb-83f8-02d5f014588f.png)

**Comment by danilovictor** [comment]
me da uma sugestão ou uma solução , sempre que eu faço algum projeto nele , quando eu faço uma build sempre da isso

**Reply by ninetails** [reply]
Tenta ver se isso satisfaz, alterar o `file-loader` com as fontes:

https://dev.to/varya/using-web-fonts-with-sass-and-webpack-51m4
